config/runtime/server.go

```diff
@@ -420,6 +420,7 @@ func newInlineBackend(
 		if diags.HasErrors() {
 			return nil, diags
 		}
+		bodies = append(bodies, inlineDef.Body())
```

Comment:
Can you please explain the reason why this must be added here? `backendConf` gets created from `inlineBlock` which refers to `inlineDef` and gets added to `bodies` one line below.

Reply:
The `inlineDef` is the `endpoint` itself. The `inlineBlock` is the `backend` block inside the `endpoint`. This block refers to the `definitions.backend` block. So we have the body from the `definitions.backend` and we add the body from the `backend` block inside the `endpoint` in the next line, but the body of the `endpoint` is missing.

If you remove this line, the test fails, because all of `ae_*` params are missing.

Reply:
I have rebuild the setup to return the `err` instead of fatal log. The failed tests are related to this line. If I remove this one, just the add query params are failing.